automated-comments/go/two_fer/use_fmt_placeholder_for_string.md

```diff
@@ -0,0 +1,5 @@
+Tip: any type of value in Go can be printed using the format character `%v`. However, there's also a different format for each type (for example `%s` for strings). The type specific formats also offer more options for formatting that type. For example the format `%f`, for `float` values, can also specify the precision: `%.2f`.
+
+Additionally, if you use the type specific placeholders and you refactor your code you will get notified 
+if you changed the type of the variable but not the type of the placeholder. This can e.g. help 
+to avoid breaking a string based api without noticing it.
```

Comment:
I'm not sure what the actual recommendation is here.

Reply:
To rather use type specific identifiers than the general `%v`. If e.g. the type of the variable changes during some refactoring, `%v` might silently "fail" while a type specific placeholder will alarm you (or better the linters will).

Reply:
Ah! That's great. Let me see if I can try rephrasing based on that...

> In Go people tend to prefer type-specific placeholders rather than the generic `%v`. There are two reasons for this. First, more specific placeholders often have additional options, for example the format `%f` for `float` values can also specify the precision: `%.2f`. The second reason is that `%v` accepts anything. If a variable type changes during a refactoring, `%v` will print the default for that type, which might not be the desired behavior. A type-specific placeholder will cause the linter to complain, catching the error before it gets deployed to production.

I don't know. Suggestions/improvements welcome :-)

Reply:
I like it. Updated and replaced `anything` with `any value type`.